test/testUtils/FrontEndTestHelper.scala

```diff
@@ -83,6 +85,18 @@ trait FrontEndTestHelper extends PlaySpec with MockitoSugar with Injecting with
     }
   }
 
+  def getConfig(blockAdditionalMetadata: Boolean): Configuration = {
+    val config: Map[String, ConfigValue] = ConfigFactory
+      .load()
+      .withValue("featureAccessBlock.closureMetadata", ConfigValueFactory.fromAnyRef(blockAdditionalMetadata.toString))
+      .withValue("featureAccessBlock.descriptiveMetadata", ConfigValueFactory.fromAnyRef(blockAdditionalMetadata.toString))
+      .entrySet()
```

Comment:
I'm not sure if we already have test scenarios to test if `featureAccessBlock.closureMetadata` is true but `featureAccessBlock.descriptiveMetadata` is false or other way around. So in that case, the controllers (`TransferAgreementComplianceController` and `TransferAgreementPrivateBetaController`) should show the `transferAgreementForm`.

Reply:
Ah good point yeah. I've added in a table with each of the options for the two booleans and run the tests for all of them. It's in a loop so when we get rid of the feature block, we can just remove the outer loop.